src/Utils/StringUtils.php

```diff
@@ -28,4 +28,41 @@ public static function csvToArray($args)
         //
         return array_map('trim', array_filter(explode(',', is_array($args) ? implode(',', $args) : $args)));
     }
+
+    /**
+     * Replace placeholders in a string.
```

Comment:
lol np. I'll go back and annotate the other PR too. I often procrastinate on this when parameters are in flux, but that can lead to never doing it.